doap.ttl

```diff
@@ -34,15 +33,15 @@
 .
 
 <https://mastodon.social/users/steveblackmon> a as:Person ;
-   as:id "https://mastodon.social/@steveblackmon" ;
-   as:name "Steve Blackmon"@en ;
-   vcard:given-name "Steve"@en ;
-   vcard:family-name "Blackmon"@en ;
+   as:name "Steve Blackmon" ;
+   vcard:given-name "Steve" ;
+   vcard:family-name "Blackmon" ;
    vcard:email "sblackmon@apache.org" ;
-   as:summary "Techie, Dad, Salesforce, PMC Chair of http://streams.apache.org"@en ;
-   as:preferredUsername "steveblackmon"@en ;
+   as:summary "Techie, Dad, Salesforce, PMC Chair of http://streams.apache.org" ;
+   as:preferredUsername "steveblackmon" ;
```

Comment:
these (optional) language annotations complicate json-ld serialization, so removing for now